exercise3/submissions/README.md

```diff
@@ -0,0 +1,14 @@
+Exercise 3 Notes
+================
+
+*Output files are written to the output directory and have a _comp extension.
+For instance output.json is replicated as output_comp.csv in the output directory
+
+*Testing.  Testing for this is also included in the driver file.
+The output file isn't comparable to the example output for a few reasons.
+1)  The output file wasn't real generated data.  It appeated to be real data for one report that was simply replicated for all reports to give an example file.
+2)  json format isn't guaranteed to be sorted.
+
```

Comment:
Ah yes.  I completely forgot about OrderedDicts.   Most of the work I've done with JSON files in the past has been primarily to ship data around between systems.  For example at Addgene we provided JSON data downloads that other companies would use to populate their databases.  In those cases the order wouldn't be that important as humans likely didn't have to read it. 

In any event, I updated the code to use OrderedDicts and verified that that output JSON is now in the correct order.